### ISSUE TITLE
App Can't Read Sflix.to's Hydrax Server

### ISSUE THREAD

**Issue by noah010** [LagradOst/CloudStream-3#1325]
### Source information

sflix.to

### Source link

https://sflix.to/

### Steps to reproduce

1. Search for "Extraordinary Attorney Woo" in the Cloudstream app.
2. Choose the poster from sflix.to.
3. Click Episode 5.
4. A popup window stating "No links Found" will appear
5. Now, visit this link "https://sflix.to/watch-tv/free-extraordinary-attorney-woo-hd-83917.8783886".
6. As you can see, it only has one server, which is Hydrax, as opposed to the prior episodes, which had servers like Upcloud and Vidcloud, the only ones, in my opinion, that the app was reading.

![image](https://user-images.githubusercontent.com/74418582/180398863-8a511b93-36fb-4a42-b89e-5bfc134bc51b.png)
![image](https://user-images.githubusercontent.com/74418582/180399081-709e2624-faa5-458b-b22e-d2d8494280c1.png)
![image](https://user-images.githubusercontent.com/74418582/180399192-ee19c575-8ec4-4033-afbd-2576dfb9290e.png)

### CloudStream version

3.0.0

### Android version

Android 11

### Other details

It also occurs in other television shows I watch, notably "Money Heist: Korea - Joint Economic Area" S01E04. Same provider, android version, and app version. This problem might, hopefully, be resolved. Thank you!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/LagradOst/CloudStream-3/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

**Comment by KillerDogeEmpire** [comment]
@noah010 that's because there is no hydrax Extractor

**Comment by noah010** [comment]
Do you mean that the specific episode of the series that only has a hydrax server cannot be played on cloudstream?

**Comment by KillerDogeEmpire** [comment]
@

> Do you mean that the specific episode of the series that only has a hydrax server cannot be played on cloudstream?

Yes, not until hydrax gets added as an extractor

**Comment by noah010** [comment]
Do you suppose the app could get a hydrax extractor in the future?

**Comment by KillerDogeEmpire** [comment]
Yes I do, in the future
